tensorflow/lite/schema/schema_generated.h

```diff
@@ -657,6 +657,10 @@ struct StableHLOCompositeOptions;
 struct StableHLOCompositeOptionsBuilder;
 struct StableHLOCompositeOptionsT;
 
+struct StablehloShiftLeftOptions;
+struct StablehloShiftLeftOptionsBuilder;
```

Comment:
I don't think we want to include the changes to the generated file here. Please remove.